### PR TITLE
Decred Integration

### PR DESCRIPTION
**Pull request by KhalilBellakrid**

### CODE REVIEW

File: packages/hw-app-btc/src/Btc.js [hunk 1]
```diff
@@ -136,7 +136,8 @@ export default class Btc {
 
   getTrustedInput(
     indexLookup: number,
-    transaction: Transaction
+    transaction: Transaction,
+    isDecred?: boolean = false
```
Comment (gre) [inline packages/hw-app-btc/src/Btc.js]: Prefer to pass additionals here and look for decred string because we've seen this doesn't scale

Comment (KhalilBellakrid) [reply packages/hw-app-btc/src/Btc.js]: I agree ! I made requires changes

**Comment by gre** [comment]
you might want to run `yarn prettier`

**Comment by gre** [comment]
ok nevermind, test pass

File: packages/hw-app-btc/src/Btc.js [hunk 2]
```diff
@@ -288,7 +310,7 @@ export default class Btc {
     inputs: Array<{ trustedInput: boolean, value: Buffer }>,
     bip143?: boolean = false,
     overwinter?: boolean = false,
-    additionals: string[] = []
+    additionals?: Array<string>
```
Comment (gre) [inline packages/hw-app-btc/src/Btc.js]: i didn't see this in previous review, but why not keeping like before?

`additionals: string[] = []` guarantee the array will be defaulted to `[]` when not provided. any reason to switch to `?: ` again?

**Pull request merged**